### ISSUE TITLE
Line break in function block with FBinit call 

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#20]
```
// in
PROGRAM MAIN
VAR
    fbTest : Test();
END_VAR

// out
PROGRAM MAIN
VAR
    fbTest : 
    Test();
END_VAR

// expected
PROGRAM MAIN
VAR
    fbTest : Test();
END_VAR
```

The `()` are not necessary, but they can be present.

- STweep v2.0.1
- [settings.txt](https://github.com/Barteling/STweep/files/5918990/settings.txt)

**Comment by Barteling** [comment]
Bug confirmed, 

With an argument it works as expected:
<pre>
PROGRAM MAIN
VAR
    fbTest : Test(A := B);
END_VAR
</pre>

With a bit of luck if have it fixed before tonights planned release.

**Comment by Barteling** [comment]
Fixed in STweep for TwinCAT 2.1.0 https://www.stweep.com/releases/
Fixed in STweep for Codesys 2.1.0
